simulation/traffic_simulator/src/helper/ostream_helpers.cpp

```diff
@@ -0,0 +1,130 @@
+// Copyright 2015 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <traffic_simulator/helper/ostream_helpers.hpp>
+
+namespace traffic_simulator
+{
+// basic ros types
+std::ostream & operator<<(std::ostream & os, const geometry_msgs::msg::Point & point)
+{
+  os << "Point(x: " << point.x << ", y: " << point.y << ", z: " << point.z << ")";
+  return os;
+}
+
+std::ostream & operator<<(std::ostream & os, const geometry_msgs::msg::Vector3 & vector)
+{
+  os << "Vector3(x: " << vector.x << ", y: " << vector.y << ", z: " << vector.z << ")";
+  return os;
+}
+
+std::ostream & operator<<(std::ostream & os, const geometry_msgs::msg::Quaternion & quat)
+{
+  os << "Quaternion(x: " << quat.x << ", y: " << quat.y << ", z: " << quat.z << ", w: " << quat.w
+     << ")";
+  return os;
+}
+
+std::ostream & operator<<(std::ostream & os, const geometry_msgs::msg::Pose & pose)
+{
+  os << "Pose(position: " << pose.position << ", orientation: " << pose.orientation;
+  return os;
+}
```

Comment:
fixed missing ")" and structured

```suggestion
std::ostream & operator<<(std::ostream & os, const geometry_msgs::msg::Pose & pose)
{
  os << "Pose(\n";
  os << "  position: " << pose.position << "\n";
  os << "  orientation: " << pose.orientation << "\n";
  os << ")";
  return os;
}
```